libs/pages/application/src/lib/ui/import-environment-variable-modal/import-environment-variable-modal.tsx

```diff
@@ -106,6 +106,7 @@ export function ImportEnvironmentVariableModal(props: ImportEnvironmentVariableM
                 <Controller
                   name={key + '_key'}
                   control={control}
+                  defaultValue=""
```

Comment:
defaultValue like this is finally need everywhere or is it just an oversight?

Reply:
These one can be removed